<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>723</width>
    <height>478</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget
{
	//background-color: rgb(252, 252, 252);
}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QGroupBox" name="botsCointainer">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="title">
       <string>Bots</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTabWidget" name="gameInfoContainer">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="currentIndex">
       <number>3</number>
      </property>
      <widget class="QWidget" name="mapTab">
       <attribute name="title">
        <string>Map</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="actionsTab">
       <attribute name="title">
        <string>Actions</string>
       </attribute>
       <widget class="QPushButton" name="btnAttack">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>40</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Attack</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btnPickup">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>90</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Pickup</string>
        </property>
       </widget>
       <widget class="QLabel" name="lblBotInstance">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>190</y>
          <width>101</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>no bot instance</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="inventoryTab">
       <attribute name="title">
        <string>Inventory</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="skillsTab">
       <attribute name="title">
        <string>Skills</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QGridLayout" name="skillsLayout"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="scriptsTab">
       <attribute name="title">
        <string>Scripts</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>723</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionAttach"/>
    <addaction name="actionOptions"/>
   </widget>
   <addaction name="menuMenu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAttach">
   <property name="text">
    <string>Attach</string>
   </property>
  </action>
  <action name="actionOptions">
   <property name="text">
    <string>Options</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
